### ISSUE TITLE
error: (-215) intersection.size() <= 8 in function rotatedRectangleIntersection

### ISSUE THREAD

**Issue by clw5180** [DetectionTeamUCAS/RetinaNet_Tensorflow_Rotation#3]
训练过程中会随机出现这个错误，有的时候训练了200000step才出现，有时候很快就出现，我在github的opencv项目下看到也有其他人提到了这个问题，说是可以在intersection.cpp里面把float改成double，但是我并没有找到这个文件；另外我把opencv3.4.2版本给conda remove了，不知道程序为什么还能跑起来，也没报No module named 'cv2'的错误，不知道是什么原因。有谁遇到了这个问题么？

**Comment by bigKoki** [comment]
能问你一个问题吗，我训练的时候会报错。。。/beta not found in ...../resnet101_v1d.ckpt,请问您有遇到这个问题吗？

**Comment by yangxue0827** [comment]
换个版本号的名字试试，在cfgs里 @864546664

**Comment by bigKoki** [comment]
> 换个版本号的名字试试，在cfgs里 @864546664

Tensor name "resnet101_v1d/C1/conv0/BatchNorm/beta" not found in checkpoint files /home/dl/桌面/RetinaNet_Tensorflow_Rotation-master/data/pretrained_weights/resnet101_v1d.ckpt
作者您好，我遇到的问题是这个，请问你指的版本号是Version吗？请问一下该如何改动，多谢了！！！

**Comment by yangxue0827** [comment]
那就不是版本号的问题了，你有修改过什么地方吗，或者你重新clone一份代码试试。 @864546664

**Comment by bigKoki** [comment]
> 那就不是版本号的问题了，你有修改过什么地方吗，或者你重新clone一份代码试试。 @864546664
训练的代码我并没有改动，生成裁减图片的代码我有些许改动使之成功运行了，但是报的错误和数据集无关，我先试试你的办法，多谢楼主

**Comment by yangxue0827** [comment]
那你重新下载一下权重试试 @864546664

**Comment by clw5180** [comment]
看来作者大佬也不知道,默默关掉好了...

**Comment by yangxue0827** [comment]
你这个错误是遇到了两个重合的框导致算交点的时候有非常多，而这个函数不支持这种情况的计算。这个保存我只在裁图测试的时候遇到过，训练我真的不清楚为什么会有这种情况。 @clw5180

**Comment by yangxue0827** [comment]
![image](https://user-images.githubusercontent.com/29257168/61880296-36d49e80-af27-11e9-8fb3-248492bef10b.png)
因为除了完全重合有无数个角点，其余情况最多只有8个角点 @clw5180

**Comment by clw5180** [comment]
> 你这个错误是遇到了两个重合的框导致算交点的时候有非常多，而这个函数不支持这种情况的计算。这个保存我只在裁图测试的时候遇到过，训练我真的不清楚为什么会有这种情况。 @clw5180

感谢指点！

**Comment by ZhenyF** [comment]
大佬，请问这个R-IoU的截图出自哪里？
